spec/inputs/pipe_spec.rb

```diff
@@ -1,43 +1,63 @@
 # encoding: utf-8
-require "logstash/devutils/rspec/spec_helper"
+require_relative "../spec_helper"
```

Comment:
Not sure this change feels right. It saves us one line of code here, but moves it to another file and requires a new file to be maintained. Can you tell me about the gain here?